src/mscorlib/shared/System/UnitySerializationHolder.cs

```diff
@@ -12,7 +12,6 @@
 
 namespace System
 {
-    [Serializable]
     // Holds classes (Empty, Null, Missing) for which we guarantee that there is only ever one instance of.
```

Comment:
I'll include that with general ISerializable cleanup (I've updated https://github.com/dotnet/corefx/issues/19119 to include that)